packages/google-auth/src/service/authed-request.service.ts

```diff
@@ -53,10 +53,9 @@ export class AuthedRequest {
           private_key: process.env.GCP_PRIVATE_KEY.replace(/\\n/g, '\n'),
         },
       }),
-      scopes: ['https://www.googleapis.com/auth/cloud-platform'],
     })
 
-    const client = googleAuth.getClient() as unknown as JWT | Compute
+    const client = (await googleAuth.getClient()) as unknown as JWT | Compute
```

Comment:
Com esse **await** não precisa mais do `unknown`
```suggestion
    const client = (await googleAuth.getClient()) as JWT | Compute
```